Rework the quickstart cli guide (#147)
rancher/elemental-docs: docs/quickstart-cli.md

## Preparing the installation (seed) image

> This is the last step, you need an Elemental Teal seed image that includes automatically (not for aarch64 yet) the initial registration config, so it can be auto registered, installed and fully deployed as part of your cluster.

Reviewer: ```suggestion
This is the last step: You need an Elemental Teal seed image that includes automatically (not for aarch64 yet) the initial registration config, so it can be auto registered, installed and fully deployed as part of your cluster.
```

Reviewer: What's that aarch64 remark about ?

Author: It seems to me the idea is that it is not `automatic` for aarch64 🤷🏼 
I am going to drop that remark and also remove *automatically*...

```
:::

> The contents of the registration config file are nothing  more than the registration URL that the node needs to register, the proper server certificate and few options for the registration process, so it can connect securely.

Reviewer 2: ```suggestion
The contents of the registration config file are nothing more than the registration URL that the node needs to register, the proper server certificate and few options for the registration process, so it can connect securely.
```

### Selecting the right machines to join a cluster

> In order for the `MachineInventorySelectorTemplate` to select the nodes, an *elemental* label to the `MachineInventory` is needed.

Reviewer: ```suggestion
In order for the `MachineInventorySelectorTemplate` to select the nodes, adding an *elemental* label to the `MachineInventory` is needed.
```

We have added the *elemental*:*fire* label in the `MachineRegistration` `machineInventoryLabels` map, so all the `MachineInventories` originated from it already have the label.
One could anyway skip the label from the `MachineRegistration` and add it later:

```shell showLineNumbers
kubectl -n fleet-default label machineinventory $(kubectl get machineinventory -n fleet-default --no-headers -o custom-columns=":metadata.name") element=fire
```

> As soon as `MachineInventories` with the *element*:*fire* are present, the corresponding machines will auto-deploy the cluster via the chosen provider (k3s/rke).

Reviewer: ```suggestion
As soon as `MachineInventory`s with the *element*:*fire* are present, the corresponding machines will auto-deploy the cluster via the chosen provider (k3s/rke).
```

After a few minutes your new cluster will be fully provisioned!!

## How can I choose the kubernetes version and deployer for the cluster?

In your cluster.yaml file there is a key in the `Spec` called `kubernetesVersion`. That sets the version and deployer that will be used for the cluster,

> for example for rke `v1.24.8` while for rke2 would be `v1.24.8+rke2r1` and for k3s `v1.24.8+k3s1`.

Reviewer: ```suggestion
for example Kubernetes`v1.24.8` for rke2 would be `v1.24.8+rke2r1` and for k3s `v1.24.8+k3s1`.
```
